icu4j/main/collate/src/test/java/com/ibm/icu/dev/test/util/ULocaleCollationTest.java

```diff
@@ -242,25 +242,25 @@ void checkService(String requestedLocale, ServiceFacade svc,
     public void TestNameList() {
         String[][][] tests = {
                 /* name in French, name in self, minimized, modified */
-//                {{"fr-Cyrl-BE", "fr-Cyrl-CA"},
-//                    {"Français (cyrillique, Belgique)", "Français (cyrillique, Belgique)", "fr_Cyrl_BE", "fr_Cyrl_BE"},
-//                    {"Français (cyrillique, Canada)", "Français (cyrillique, Canada)", "fr_Cyrl_CA", "fr_Cyrl_CA"},
-//                },
+                {{"fr-Cyrl-BE", "fr-Cyrl-CA"},
+                    {"Français (cyrillique, Belgique)", "French (Cyrillic, Belgium)", "fr_Cyrl_BE", "fr_Cyrl_BE"},
```

Comment:
Hardcoding the English results and forcing the default locale to be English seemed like the path of least resistance, so I took it.  TBH, I didn't think about using an empty string (or a null) as an indicator to skip the test altogether.  I kind of like that, but then you're not testing the "in self" case at all, whereas forcing the default locale to be English does test that logic.  On the other hand, getting English in these cases feels really wrong...

I'm inclined to just merge this and call it a day until some other problem comes up or somebody suggests a better change to the underlying code.  It sounds like you don't object to that, but I'll leave this PR open until at least the end of the day just in case you change your mind or somebody else weighs in.

Reply:
> I'm inclined to just merge this and call it a day

+1